lib/switchboard.rb

```diff
@@ -20,5 +14,13 @@ def lookup(id)
     def base_url
       ENV['SWITCHBOARD_BASE_URL']
     end
+
+    def connection
```

Comment:
The approach we've taken with Search is to pull the connection from a registry - https://github.com/guidance-guarantee-programme/pension_guidance/blob/master/config/initializers/search.rb#L1-L2